docs/community/sdk-tiers.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: SDK Tiering System
+description: Feature completeness, protocol support, and maintenance commitment levels for Model Context Protocol SDKs
+---
+
+The MCP SDK Tiering System establishes clear expectations for feature completeness, protocol support, and maintenance commitment across official and community SDKs. This helps developers choose the right SDK for their needs and provides SDK maintainers with a clear path for improvement.
+
+## Overview
+
+SDKs are classified into three tiers based on feature completeness, maintenance commitment, and documentation quality:
+
+- **Tier 1**: Fully supported SDKs with complete protocol implementation
+- **Tier 2**: Well-maintained SDKs working toward full support
+- **Tier 3**: Experimental or specialized SDKs
+
+## Tier Requirements
+
+| Requirement                 | Tier 1: Fully Supported                                                          | Tier 2: Commitment to Full Support                               | Tier 3: Experimental   |
+| --------------------------- | -------------------------------------------------------------------------------- | ---------------------------------------------------------------- | ---------------------- |
+| **Conformance Tests**       | 100% pass rate                                                                   | 80% pass rate                                                    | No minimum             |
+| **New Protocol Features**   | Within 2 weeks of Release Candidate release, before the new spec version release | Within 6 months                                                  | No timeline commitment |
+| **Issue Triage**            | Within 2 business days                                                           | Active issue tracking                                            | No requirement         |
+| **Critical Bug Resolution** | Within 7 days                                                                    | Active management                                                | No requirement         |
+| **Stable Release**          | Required with clear versioning                                                   | At least one stable release                                      | Not required           |
```

Comment:
Do we want to bind this to protocol spec versions (i.e., one stable release per spec version)? Maybe not, since some SDKs would be playing catch-up anyway. Maybe instead we focus on regularly updated stable releases?